api/api/src/test/scala/uk/ac/wellcome/platform/api/services/ElasticsearchServiceTest.scala

```diff
@@ -711,7 +712,7 @@ class ElasticsearchServiceTest
   }
 
   private def createItemWithLocationType(
-    locationType: LocationType): Identified[Item] =
+    locationType: LocationType): Item[Minted] =
```

Comment:
Why `Minted` and not `Identified` here?

Reply:
I think I did it like this as we are generally parameterising by `Minted` / `Unminted` in other declarations, rather than the concrete types. We never use these as an `Item[Identified]` (only as `Item[Minted]`), so being consistent should hopefully make the code more predictable on the whole.